lang/guile-3.0/Portfile

```diff
@@ -0,0 +1,155 @@
+
+description         GNU's Ubiquitous Intelligent Language for Extension (guile)
+long_description    Guile is an interpreter for the Scheme programming \
+                    language, packaged for use in a wide variety of \
+                    environments. Guile implements Scheme as described in \
+                    the Revised^5 Report on the Algorithmic Language Scheme \
+                    (usually known as R5RS), providing clean and general \
+                    data and control structures. \
+                    Guile goes beyond the rather austere language presented \
+                    in R5RS, extending it with a module system, full access \
+                    to POSIX system calls, networking support, multiple \
+                    threads, dynamic linking, a foreign function call \
+                    interface, powerful string processing, and many other \
+                    features needed for programming in the real world.
+homepage            https://www.gnu.org/software/guile/guile.html
+
+master_sites        gnu:${my_name}
+distname            ${my_name}-${version}
+dist_subdir         ${my_name}
+use_xz              yes
+
+checksums           rmd160  50534bf521961d08fb1b4fb6e1d5dbefe305a4f3 \
+                    sha256  1a2625ac72b2366e95792f3fe758fd2df775b4044a90a4a9787326e66c0d750d \
+                    size    5750636
+
+compiler.c_standard 2011
+
+# https://trac.macports.org/ticket/59365
+xcode_workaround.fixed_xcode_version 11.2
+xcode_workaround.type avoid_xcode_compiler
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:gettext \
+                    port:texinfo
+
+# fixes: sed: -i may not be used with stdin
+depends_build-append \
+                    port:gsed
+configure.env-append \
+                    SED=${prefix}/bin/gsed
+build.env-append \
+                    SED=${prefix}/bin/gsed
+
+# guile-config is a wrapper around pkg-config
+depends_lib-append \
+                    port:boehmgc \
+                    port:gettext-runtime \
+                    port:gmp \
+                    port:libffi \
+                    port:libiconv \
+                    port:libtool \
+                    port:libunistring \
+                    port:ncurses \
+                    port:pkgconfig \
+                    port:readline
+
+configure.args-append \
+                    CPPFLAGS="-I${prefix}/include" \
+                    LDFLAGS="-L${prefix}/lib" \
+                    --disable-error-on-warning \
+                    --disable-silent-rules \
+                    --enable-regex \
+                    --enable-shared \
+                    --infodir="${prefix}/share/info" \
+                    --mandir="${prefix}/share/man" \
+                    --program-suffix=-${binding_ver}
+
+# configure may err on version mismatch otherwise
+patchfiles-append   patch-automake.diff
+
+if {${build_arch} in [list i386 ppc]} {
```

Comment:
configure.build_arch

Reply:
@kencu I'm wondering that is the differences between `configure.build_arch` and just `build_arch`?

Reply:
> @kencu I'm wondering that is the differences between `configure.build_arch` and just `build_arch`?

@MarcusCalhoun-Lopez provided a great explanation, via his recent PR to fix these:

https://github.com/macports/macports-ports/pull/18857#issuecomment-1566212319